arelle/CntlrCmdLine.py

```diff
@@ -626,6 +639,27 @@ def _pluginHasCliOptions(moduleInfo):
     return False
 
 
+def _parseRuntimeOptionsArg(arg: str) -> Any:
+    """
+    Parses a runtime options argument which may be either a JSON string or a file path to a JSON file.
+    :param arg: Raw JSON or a path to a .json file.
+    :return: Runtime options dictionary, or None if parsing failed.
+    """
+    try:
+        return json.loads(arg)
+    except ValueError as e:
+        # Argument is not valid JSON
+        pass
+    if os.path.exists(arg):
+        try:
+            with open(arg) as f:
+                return json.load(f)
+        except ValueError as e:
+            # File does not contain valid JSON
```

Comment:
It seems like it would be preferable to print the actual exception (which probably has line/col info?) rather than the vague "Options argument is not a valid JSON or a path to a valid JSON file.".

Related, suggest `if json.startswith('{'): return json.loads else: with open(arg) as f: return json.load(f)` so that whether or not a file is opened is deterministic rather than sending a long/broken JSON string to a the OS to open as a filename.  (It might be even better to just have separate options for literal vs file.)